server/src/config/google-oauth.ts

```diff
@@ -0,0 +1,34 @@
+import passport from "passport";
+import { Strategy as GoogleStrategy } from "passport-google-oauth20";
+import { User } from "../model/User";
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("dotenv").config();
+
+passport.use(
+  new GoogleStrategy(
+    {
+      clientID: process.env.GOOGLE_CLIENT_ID!,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET!,
+      callbackURL: "/api/auth/google/redirect",
+    },
+    (_accessToken, _refreshToken, profile, done) => {
+      User.findOne({ googleId: profile.id }).then((currentUser) => {
+        if (currentUser) {
+          done(null, currentUser);
+        } else {
+          new User({
+            googleId: profile.id,
+            username: profile.displayName,
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
here as well

Reply:
this one is required to disable eslint for 1 line